src/Common/tests/System.Net/Sockets/Performance/SocketPerformanceTests.cs

```diff
@@ -8,7 +8,7 @@
 using Xunit;
 using Xunit.Abstractions;
 
-namespace System.Net.Sockets.Performance.Tests
+namespace System.Net.Sockets.Tests
```

Comment:
FWIW, There was a discussion about the NAMESPACES to test classes and methods.  It was agreed that ALL namespaces for test classes and methods would be <namespace of classes/methods under test>.Tests

So, for example, the HttpWebRequest API which is under the System.Net namespace but yet packaged in the System.Net.Requests namespace would have the test classes and methods under the "System.Net.Tests" namespace.

See agreement in #2898.

And System.Net.Requests changes in #3660.

Note:  The test binary name still can be named System.Net.Sockets.Unit.Tests.dll or System.Net.Sockets.Performance.Tests.dll.

cc: @stephentoub